### DISCUSSION TITLE
How to change the logo to another builtin logo ?

### DISCUSSION THREAD

**Discussion by sbera77** [fastfetch-cli/fastfetch#914] [Q&A] [answered]
I dont like how the current ubuntu logo looks and wanted to change it to the ubuntu_old builtin logo. I cant figure out how to do it

```  
"logo" : {
    "type": "builtin",
    "source": "ubuntu_old"
}
```

Should I just use an image ?

**Comment by CarterLi** [comment, marked as answer]
Removing `"type": "builtin"` should work

**Reply by sbera77** [reply]
It did , thank you
